exaudfclient/docs/script_options_requirments.md

```diff
@@ -0,0 +1,376 @@
+
+### Duplicate Options Management
+`req~multiple-options-management~1`
+
+The parser must collect multiple Script Options with the same key and same/different value. Note: the specific handling depends on the option handler.
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+### Script Option Removal
+`req~script-option-removal~1`
+
+The parser handler must remove found known Script Options from the original script code. 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+- `feat~java-specific-script-options~1`
+
+### Script Option Unknown Options Behavior V1
+`req~script-option-unknown-options-behvaior-v1~1`
+
+Unknown script options must remain untouched in the script code. The Java compiler is supposed to throw an error message during the compilation. 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+- `feat~java-specific-script-options~1`
+
+Tags: V1
+
+### Script Option Unknown Options Behavior V2
+`req~script-option-unknown-options-behvaior-v2~1`
+
+The parser handler must detect unknown script options and throw an exception if such an options is found. 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+- `feat~java-specific-script-options~1`
+
+Tags: V2
+
+### Escape Sequence Script Options Parsing
+`req~escape-sequence-script-options-parsing~1`
+
+The following rules for escape sequences at any place within a script optionValue are to be applied:
+- '\n' => <line feed> character
+- '\r' => <carriage return> character
+- '\;' => ';' character
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+
+### Java %scriptclass Option Handling V1
+`req~java-scriptclass-option-handling-v1~1`
+
+The Java parser handler must correctly identify the first `%scriptclass` option and remove only this single instance from the script code. Any further occurrences of `%scriptclass` option shall stay in the source script code.
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+
+### Java %scriptclass Option Handling V2
+`req~java-scriptclass-option-handling-v2~1`
+
+The Java parser handler must correctly identify the first `%scriptclass` option and remove any additional occurrences of this option within the script code.
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Handling Multiple Options
+`req~java-jar-option-handling-multiple-options~1`
+
+The Java parser handler must find multiple `%jar` options. The values are to be interpreted as the Java CLASSPATH: `<file1>:<file2>:...:<filen>`. The Java parser handler shall split the entries by the colon character.
```

Comment:
Maybe add note to source code which defines how colon is handled in the jvm

Reply:
Done